### ISSUE TITLE
Was Assigned a issue . Created a PR

### ISSUE THREAD

**Issue by sonalsingh25** [HarshCasper/NeoAlgo#2875]
### Please note that this issue tracker is not a help form and this issue will be closed.
I created a issue for Greedy_Algorithms in java. I was assigned to it by @gat786. Started working on it only after being ASSIGNED.  Today @HarshCasper closed my PR stating that "We don't accept PR'S without any issue"
I would request @gat786 to please look into it . This was my first PR and also my first time in open-source. Please guide me if I'm wrong.
.
![Kruskal's Minimum Spanning Tree Algorithm · Issue #2641 · TesseractCoding_NeoAlgo - Google Chrome 12-03-2021 19_35_52](https://user-images.githubusercontent.com/73117995/110951404-072c1180-836b-11eb-8299-ef4112345664.png)
![Kruskal's Minimum Spanning Tree Algorithm · Issue #2641 · TesseractCoding_NeoAlgo - Google Chrome 12-03-2021 19_35_52](https://user-images.githubusercontent.com/73117995/110951445-11e6a680-836b-11eb-8f29-22cb0df690b1.png)

Please contact us instead.

- [Website](https://www.tesseractcoding.tech/)
- [Email](team@tesseractcoding.tech)
- [GitHub](https://github.com/TesseractCoding)
- [Instagram](https://www.instagram.com/tesseractcoding/)
- [LinkedIn](https://www.linkedin.com/company/tesseract-coding/)
- [Medium](https://medium.com/tesseract-coding)
- [Telegram](https://t.me/tesseractcoding)
- [YouTube](https://www.youtube.com/c/TesseractCoding/)

**Comment by gat786** [comment]
Did you mention the issue number in the PR that you had raised? 
The PR can opened again that is not such a big issue.

**Comment by gat786** [comment]
Without a proper link its very difficult to navigate and search for issues. That's why we ask for the link of the issue in PR which is raised.

**Comment by HarshCasper** [comment]
Hi @sonal251001 

Thanks for your Issue! 

Let's reiterate a few things that every contributor should follow: 

- They should have an Issue assigned to them. 
- They should make a PR with the **Issue linked to it** 
- They should address the reviews sent by the maintainer 
- They should be nice and respectful during the whole period 

Remember that if any of the steps are skipped at any time, the PR would be closed aptly. You did not add the Issue link on your PR. 

Now the question begs, how do we know that the Issue was assigned to you? If we take a look at the section: 

![image](https://user-images.githubusercontent.com/47351025/110968672-d43f4900-837d-11eb-94ea-509c464e6f40.png)

We have a PR template and we literally asked to add the Issue link and the above was added there. Raising an Issue first does not mean you will be necessarily the one to work on it. You were *wrong* at this point. 

If you have the Issue assigned to you, make another PR and add the link of the Issue you are assigned to. 

Thanks!
